resume: 1st pass at making it more readable

vpayno opened the pull request:
### Documentation

- Link resume.md to README.md

### Miscellaneous Tasks

- Add .gitignore
- Add spellcheck workflow
- Add markdown lint workflow
- Add yaml lint workflow
- Add prose lint workflow
- Add link check workflow
- Add github action lint workflow

### Resume

- 1st pass at making it more readable

### Tools

- Add markdown to html script

vpayno committed:
resume: 1st pass at making it more readable

- break up large paragraphs into smaller chuncks
- use lists instead of large blocks of things
- use markdown syntax

vpayno committed:
ci: add spellcheck workflow

vpayno committed:
ci: add markdown lint workflow

vpayno committed:
ci: add yaml lint workflow

vpayno committed:
ci: add prose lint workflow

vpayno committed:
ci: add link check workflow

vpayno committed:
ci: add github action lint workflow

williamh commented:
@vpayno  I updated the pr after merging one of the commits since it seemed to be the easiest to do right away. I like the style changes, but one of the actions failed when I pushed the update. If you want to fix the failure go ahead, but I'll take a look later tonight if you don't get to it.
Thanks.

vpayno committed:
chore: add .gitignore

vpayno committed:
tools: add markdown to html script

vpayno commented:
@williamh all actions are working now (fixed typos, added words to dictionary, etc.).

I also added a simple script to create an index.html file from the resume.md file. That way your website will have an index page and it's html and not, text which is what people expect.

--

nevermind, deleted the html file. Didn't notice you had a GitHub pages job that created an HTML file.


You should set the project's website to https://williamh.github.io/
